drv/stm32xx-sys/src/main.rs

```diff
@@ -61,11 +61,11 @@ where
     S: pac::RegisterSpec<Ux = u32> + pac::Readable + pac::Writable,
 {
     unsafe fn set_bit(&self, index: u8) {
-        self.modify(|r, w| unsafe { w.bits(r.bits() | 1 << index) });
+        self.modify(|r, w| w.bits(r.bits() | 1 << index));
     }
 
     unsafe fn clear_bit(&self, index: u8) {
-        self.modify(|r, w| unsafe { w.bits(r.bits() & !(1 << index)) });
+        self.modify(|r, w| w.bits(r.bits() & !(1 << index)));
```

Comment:
both of these lines are warnings that I fixed, I assume they're new. Can leave this out if someone objects for some reason.

Reply:
I also ran into this when bumping one of the PAC crates for an experiment, so I think this is fine.

Reply:
This is fine, the PAC crates occasionally mark `bits` accessors as safe (sometimes in ways that are not correct), so any time we bump the version expect to move some unsafes around.